openbas-front/src/admin/components/scenarios/scenario/tests/ScenarioTests.tsx

```diff
@@ -0,0 +1,181 @@
+import { makeStyles } from '@mui/styles';
+import React, { CSSProperties, FunctionComponent, useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { List, ListItem, ListItemButton, ListItemIcon, ListItemText } from '@mui/material';
+import type { ScenarioStore } from '../../../../../actions/scenarios/Scenario';
+import { useFormatter } from '../../../../../components/i18n';
+import ItemStatus from '../../../../../components/ItemStatus';
+import type { InjectTestStatus, SearchPaginationInput } from '../../../../../utils/api-types';
+import { initSorting } from '../../../../../components/common/pagination/Page';
+import { fetchInjectTestStatus, searchScenarioInjectTests } from '../../../../../actions/inject_test/inject-test-actions';
+import SortHeadersComponent from '../../../../../components/common/pagination/SortHeadersComponent';
+import InjectIcon from '../../../common/injects/InjectIcon';
+import { isNotEmptyField } from '../../../../../utils/utils';
+import Empty from '../../../../../components/Empty';
+import InjectTestDetail from '../../../injects/InjectTestDetail';
+
+const useStyles = makeStyles(() => ({
+  bodyItems: {
+    display: 'flex',
+    alignItems: 'center',
+  },
+  bodyItem: {
+    height: 20,
+    fontSize: 13,
+    whiteSpace: 'nowrap',
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    paddingRight: 10,
+  },
+  itemHead: {
+    paddingLeft: 10,
+    marginBottom: 10,
+    textTransform: 'uppercase',
+    cursor: 'pointer',
+  },
+  item: {
+    paddingLeft: 10,
+    height: 50,
+  },
+}));
+
+const inlineStyles: Record<string, CSSProperties> = {
+  inject_title: {
+    width: '40%',
+    cursor: 'default',
+  },
+  tracking_sent_date: {
+    width: '40%',
+  },
+  status_name: {
+    width: '20%',
+  },
+};
+
+const ScenarioTests: FunctionComponent = () => {
+  // Standard hooks
+  const classes = useStyles();
+  const { t, fldt } = useFormatter();
+
+  const { scenarioId, statusId } = useParams() as { scenarioId: ScenarioStore['scenario_id'], statusId: InjectTestStatus['status_id'] };
+  const [selectedTest, setSelectedTest] = useState<InjectTestStatus | null>(null);
```

Comment:
You can just make: scenario Is: string, and this file are the same as exercise test